### DISCUSSION TITLE
Add Red-eyed Tree Frog (Agalychnis callidryas) 4.jpg

### DISCUSSION THREAD

**Discussion by hosttry** [BuddiesOfBudgie/budgie-backgrounds#3] [Background Suggestions]
Hi, I'd like to suggest Red-eyed Tree Frog (Agalychnis callidryas) 4.jpg to be used as a background.

Image information:
- Autor: Geoff Gallice from Gainesville, FL, USA.
- License: Creative Commons, Attribution 2.0 Generic
- Source: https://www.flickr.com/photos/dejeuxx/6360289897/

More info at :https://commons.wikimedia.org/wiki/File:Red-eyed_Tree_Frog_(Agalychnis_callidryas)_4.jpg

**Comment by serebit** [comment]
Gorgeous frog, and it satisfies most of the criteria, but unfortunately CC-BY-2.0 isn't public domain and requires attribution, so we can't add this. If you find a similar image that is explicitly marked as either public domain or Creative Commons Zero, please submit it.

**Reply by hosttry** [reply]
What about unsplash licence?
https://unsplash.com/license

Found an other photo of the same frog.
- Autor: Zdeněk Macháček
- License: Unsplash License
- Source https://unsplash.com/photos/WTr9TVVxZlQ

**Reply by serebit** [reply]
Unsplash is frustratingly close, but not quite there, since it still applies restrictions on what the images can be used for.

**Reply by serebit** [reply]
After some investigation, this image from the Wikimedia Commons appears to be public domain: https://commons.m.wikimedia.org/wiki/File:RanitaCostaRica.jpg

Doesn't look like the same species and the angle is a little awkward, but it's close.
